workers/price-reporter/Cargo.toml

```diff
@@ -18,7 +18,7 @@ tokio = { workspace = true }
 
 # === Networking === #
 tokio-stream = "0.1"
-tokio-tungstenite = "0.18"
+tokio-tungstenite = { version = "0.18", features = ["native-tls"] }
```

Comment:
this change makes sure that tls is compiled in, even when using the `price-reporter` only as a standalone library